browser/main/modals/PreferencesModal/BackupTab.js

```
@@ -0,0 +1,251 @@
+    this.props.haveToSave()
+  }
+
+  handleImportPathBrowseButtonClick (e) {
+    browseFolder(false)
+      .then((targetPath) => {
+        if (targetPath.length > 0) {
+          const { newImport } = this.state
+          newImport.path = targetPath
+          this.setState({
+            newImport
+          })
+        }
+      })
+      .catch((err) => {
+        console.error('BrowseFAILED')
+        console.error(err)
+      })
+  }
+
+  handleExportPathBrowseButtonClick (e) {
+    browseFolder(true)
+      .then((targetPath) => {
+        if (targetPath.length > 0) {
+          const { newExport } = this.state
+          newExport.path = targetPath
+          this.setState({
+            newExport
+          })
+        }
+      })
+      .catch((err) => {
+        console.error('BrowseFAILED')
+        console.error(err)
+      })
+  }
+
+  importConfig (e) {
+    if (this.state.newImport.path === '') {
+      alert(i18n.__('No file has been selected.'))
+      return false
+    }
+
+    try {
+      const externalConfig = JSON.parse(fs.readFileSync(this.state.newImport.path, 'utf-8'))
+      const { config } = this.state
+      config.hotkey = externalConfig.hotkey
+      config.ui = externalConfig.ui
+      config.editor = externalConfig.editor
+      config.preview = externalConfig.preview
+      config.blog = externalConfig.blog
+
+      this.setState({
+        config
+      })
+      this.handleSaveImport(e)
+    } catch (err) {
+      alert(i18n.__(
+        err instanceof SyntaxError ? 'File is not valid JSON.' : 'Something went wrong. Please try again'
+        ))
+      return false
+    }
+  }
+
+  exportConfig (e) {
+    if (this.state.newExport.path === '') {
+      alert(i18n.__('Please choose a directory for export file.'))
+      return false
+    }
+    fs.writeFile(
+      `${this.state.newExport.path}/boostnote.config`,
+      JSON.stringify(this.state.config), (err) => {
+        if (err) {
+          return console.log(err)
```

can you instead of just logging the error, can you alert the user that the export process has failed too please?